src/libraries/shims/fakes/README.md

```diff
@@ -0,0 +1,3 @@
+# .NETCoreApp fake assemblies
+
+The projects under this directory are fake assemblies, used by the .NETCoreApp shims (and System.Windows.Extensions) to compile type forwards against types that aren't part of the Microsoft.NETCore.App shared framework.
```

Comment:
We might want to use a different term than `fakes` since that has a different meaning already for tests.  ... maybe call them `type forward fakes` or `type forward targets`?

Reply:
I see you changed it here in the readme, but not elsewhere.  You might want to get another opinion if using the term fakes as much as you do here makes sense to others.